src/steadystateproblem.cpp

```diff
@@ -654,7 +659,8 @@ void SteadystateProblem::runSteadystateSimulation(const Solver &solver,
         /* getWrms needs to be called before getWrmsFSA such that the linear
          system is prepared for newton type convergence check */
         if (wrms_ < conv_thresh && check_sensi_conv_ &&
-            sensitivityFlag == SensitivityMethod::forward) {
+            sensitivityFlag == SensitivityMethod::forward &&
+            convergence_check_frequency % sim_steps == 0) {
```

Comment:
This will check convergence at iterations 1, 5, and 25, and never thereafter. I don't think that's intentional?! 

```suggestion
            sim_steps % convergence_check_frequency == 0) {
```